File: Source/UnitTests/TestCode/MemberMapperTest/BaseTypeMethod.java
package Test;
import junit.framework.TestCase;
public class A extends TestCase
{
	public void testAssertAreEquals()
	{
		String wp;
		String wp2;
		assertEquals(wp, wp2);
		assertEquals("message:",wp, wp2);
		assertTrue(wp.CompareTo(wp2) < 0);
	}
}

File: Source/UnitTests/TestCode/MemberMapperTest/BaseTypeMethod.cs
namespace Test
{
	using junit.framework.TestCase;
	public class A : TestCase
	{
		public void testAssertAreEquals()
		{
			String wp;
			String wp2;
			NUnit.Framework.Assert.AreEqual(wp, wp2);
			NUnit.Framework.Assert.AreEqual(wp, wp2,"message:");
			NUnit.Framework.Assert.IsTrue(wp.CompareTo(wp2) < 0);
		}
	}
}
